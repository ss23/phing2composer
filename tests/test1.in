# File format
# -----------
# Each line represents one dependent module.
#
# local_module_folder[:git_branch|:svn_revision_number] repository_url [run_dev_build=true] [local]
#
# Note that the local_module_folder can contain subfolders delimited via '/' characters
# A specific git branch or SVN revision can be added in by specifying after the local
# foldername, separated by a colon. By default, the 'master' branch of a git repository is used. 
#
# Using the "piston" flag will strip versioning information, but keep metadata in the working copy
# to update from version control later on, and merge in potential local changes.
# See http://piston.rubyforge.org for details.
#
# It is recommended to have sqlite3 and cms first with [run_dev_build] set to "false".
# Having this set to 'false' prevents the execution of the dev/build process, meaning it can be
# deferred until all dependencies are in place, specifically the framework module. List
# all additional modules after that. 
#
# Examples
#
# frontend-editing:development git://github.com/nyeholt/silverstripe-frontend-editing.git
# themes/mytheme git://local.server/themes/mytheme.git false

# cms:3.0 ssh://git@gitorious.silverstripe.com:2222/plunket/cms.git
# framework:3.0 ssh://git@gitorious.silverstripe.com:2222/plunket/framework.git
cms:3.0 https://github.com/silverstripe/silverstripe-cms.git
framework:3.0 https://github.com/silverstripe/sapphire.git
blog:master ssh://git@gitorious.silverstripe.com:2222/plunket/blog-module.git

payment:master ssh://git@gitorious.silverstripe.com:2222/plunket/payment.git
recaptcha:master ssh://git@gitorious.silverstripe.com:2222/plunket/recaptcha.git
userforms:master ssh://git@gitorious.silverstripe.com:2222/plunket/userforms.git
spamprotection:master ssh://git@gitorious.silverstripe.com:2222/plunket/spamprotection.git

fulltextsearch:master ssh://git@gitorious.silverstripe.com:2222/plunket/fulltextsearch.git
# tagfield:master ssh://git@gitorious.silverstripe.com:2222/plunket/tagfield.git
personalisation:master ssh://git@gitorious.silverstripe.com:2222/plunket/personalisation.git
googleaddressfield:master ssh://git@gitorious.silverstripe.com:2222/plunket/googleaddressfield.git 
documentconverter:master ssh://git@gitorious.silverstripe.com:2222/super-marvellous-core/documentconverter.git
widgets:master ssh://git@gitorious.silverstripe.com:2222/plunket/widgets.git
# The following are the some other modules you might like to import

sqlite3:master git://github.com/silverstripe-labs/silverstripe-sqlite3.git
# userforms:master:0.3.0 git://github.com/silverstripe/silverstripe-userforms.git
# securefiles http://svn.polemic.net.nz/silverstripe/modules/SecureFiles/tags/0.30/

translatable:master ssh://git@gitorious.silverstripe.com:2222/super-marvellous-core/translatable.git
comments:master https://github.com/silverstripe/silverstripe-comments.git


