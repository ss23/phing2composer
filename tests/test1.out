{
    "name": "name-me",
    "description": "Converted from phing to composer",
    "require": {
        "silverstripe/cms": "30.x-dev",
        "silverstripe/framework": "30.x-dev",
        "custom-silverstripe-fork/blog": "dev-master",
        "custom-silverstripe-fork/payment": "dev-master",
        "custom-silverstripe-fork/recaptcha": "dev-master",
        "custom-silverstripe-fork/userforms": "dev-master",
        "custom-silverstripe-fork/spamprotection": "dev-master",
        "custom-silverstripe-fork/fulltextsearch": "dev-master",
        "custom-silverstripe-fork/personalisation": "dev-master",
        "custom-silverstripe-fork/googleaddressfield": "dev-master",
        "custom-silverstripe-fork/documentconverter": "dev-master",
        "custom-silverstripe-fork/widgets": "dev-master",
        "silverstripe/sqlite3": "dev-master",
        "custom-silverstripe-fork/translatable": "dev-master",
        "silverstripe/comments": "dev-master"
    },
    "repositories": [
        {
            "type": "package",
            "package": {
                "name": "custom-silverstripe-fork/blog",
                "version": "dev-master",
                "type": "silverstripe-module",
                "require": {
                    "composer/installers": "*"
                },
                "source": {
                    "url": "ssh://git@gitorious.silverstripe.com:2222/plunket/blog-module.git",
                    "type": "git",
                    "reference": "master"
                }
            }
        },
        {
            "type": "package",
            "package": {
                "name": "custom-silverstripe-fork/payment",
                "version": "dev-master",
                "type": "silverstripe-module",
                "require": {
                    "composer/installers": "*"
                },
                "source": {
                    "url": "ssh://git@gitorious.silverstripe.com:2222/plunket/payment.git",
                    "type": "git",
                    "reference": "master"
                }
            }
        },
        {
            "type": "package",
            "package": {
                "name": "custom-silverstripe-fork/recaptcha",
                "version": "dev-master",
                "type": "silverstripe-module",
                "require": {
                    "composer/installers": "*"
                },
                "source": {
                    "url": "ssh://git@gitorious.silverstripe.com:2222/plunket/recaptcha.git",
                    "type": "git",
                    "reference": "master"
                }
            }
        },
        {
            "type": "package",
            "package": {
                "name": "custom-silverstripe-fork/userforms",
                "version": "dev-master",
                "type": "silverstripe-module",
                "require": {
                    "composer/installers": "*"
                },
                "source": {
                    "url": "ssh://git@gitorious.silverstripe.com:2222/plunket/userforms.git",
                    "type": "git",
                    "reference": "master"
                }
            }
        },
        {
            "type": "package",
            "package": {
                "name": "custom-silverstripe-fork/spamprotection",
                "version": "dev-master",
                "type": "silverstripe-module",
                "require": {
                    "composer/installers": "*"
                },
                "source": {
                    "url": "ssh://git@gitorious.silverstripe.com:2222/plunket/spamprotection.git",
                    "type": "git",
                    "reference": "master"
                }
            }
        },
        {
            "type": "package",
            "package": {
                "name": "custom-silverstripe-fork/fulltextsearch",
                "version": "dev-master",
                "type": "silverstripe-module",
                "require": {
                    "composer/installers": "*"
                },
                "source": {
                    "url": "ssh://git@gitorious.silverstripe.com:2222/plunket/fulltextsearch.git",
                    "type": "git",
                    "reference": "master"
                }
            }
        },
        {
            "type": "package",
            "package": {
                "name": "custom-silverstripe-fork/personalisation",
                "version": "dev-master",
                "type": "silverstripe-module",
                "require": {
                    "composer/installers": "*"
                },
                "source": {
                    "url": "ssh://git@gitorious.silverstripe.com:2222/plunket/personalisation.git",
                    "type": "git",
                    "reference": "master"
                }
            }
        },
        {
            "type": "package",
            "package": {
                "name": "custom-silverstripe-fork/googleaddressfield",
                "version": "dev-master",
                "type": "silverstripe-module",
                "require": {
                    "composer/installers": "*"
                },
                "source": {
                    "url": "ssh://git@gitorious.silverstripe.com:2222/plunket/googleaddressfield.git",
                    "type": "git",
                    "reference": "master"
                }
            }
        },
        {
            "type": "package",
            "package": {
                "name": "custom-silverstripe-fork/documentconverter",
                "version": "dev-master",
                "type": "silverstripe-module",
                "require": {
                    "composer/installers": "*"
                },
                "source": {
                    "url": "ssh://git@gitorious.silverstripe.com:2222/super-marvellous-core/documentconverter.git",
                    "type": "git",
                    "reference": "master"
                }
            }
        },
        {
            "type": "package",
            "package": {
                "name": "custom-silverstripe-fork/widgets",
                "version": "dev-master",
                "type": "silverstripe-module",
                "require": {
                    "composer/installers": "*"
                },
                "source": {
                    "url": "ssh://git@gitorious.silverstripe.com:2222/plunket/widgets.git",
                    "type": "git",
                    "reference": "master"
                }
            }
        },
        {
            "type": "package",
            "package": {
                "name": "custom-silverstripe-fork/translatable",
                "version": "dev-master",
                "type": "silverstripe-module",
                "require": {
                    "composer/installers": "*"
                },
                "source": {
                    "url": "ssh://git@gitorious.silverstripe.com:2222/super-marvellous-core/translatable.git",
                    "type": "git",
                    "reference": "master"
                }
            }
        }
    ],
    "minimum-stability": "dev"
}
